### PR TITLE
Diamond UI

### PR DESCRIPTION
**Pull request by LeonardoMolleker**
Screenshot_18
A Scaffold with a Column as a child. The Column contains three Containers, one with an AssetImage and two with a Text. One of them also wit a shadow.

### CODE REVIEW

File: lib/src/constants/constants.dart [hunk 1]
```diff
@@ -1,6 +1,9 @@
...
-  static const double width = 300.0;
+  static const double principalContainerheight = 190.0;
+  static const double principalContainerwidth = 300.0;
   static const double padding = 25.0;
-  static const double fontSize = 35.0;
+  static const double textFontSize = 35.0;
+  static const double shadowBlurRadius = 1.0;
+  static const double shadowOffsetX = 3.0;
+  static const double shadowOffsetY = 2.0;
 }
```
Comment (MarcosCGlobant) [inline lib/src/constants/constants.dart]: you are missing an extra line here, ctrl + alt + L should fix it

**Review by MarcosCGlobant — APPROVED** [review]
LGTM, just one comment of an extra line

**Review by FranciscoGaraventa — APPROVED** [review]
LG

File: lib/src/constants/constants.dart (hunk 1 above)
Comment (LeonardoMolleker) [reply lib/src/constants/constants.dart]: Fixed!

**Pull request merged**